README.md

```diff
@@ -1,38 +1,107 @@
 # GoodChat App
-GoodChat is an omni-channel chat solution that enables users to donate to us using their channel of choice (e.g. WhatsApp, Telegram, Messenger, LINE, etc.). 
+
+GoodChat is an omni-channel chat solution that enables users to donate to us using their channel of choice (e.g. WhatsApp, Telegram, Messenger, LINE, etc.).
 
 This repository is for GoodChat App which is a hybrid app that provides a user interface that allows our staff to communicate with donors as well as generate structured transactions out of these interactions in order to integrate with our existing systems.
 
 ## Getting started
+
 ### Installation
+
 Install dependencies using
+
 ```
 npm install
 ```
+
 ### Running the application
+
 ```
-npm start
+npm run dev
 ```
 
+This will start the react app as well as run graphQL code generator in the background which will watch for any code changes in `.graphql` files.
```

Comment:
For now we limit the files that we are watching to `.graphql` files.
This is in following 
1. GraphQL Code Generator Best practices (https://the-guild.dev/blog/graphql-codegen-best-practices)
and 
2. Reducing the number introspection queries to the server since if we were to watch for code changes on `tsx` files we would trigger the introspection query even if our code changes are unrelated to graphql queries.